django_prometheus/db/backends/postgresql/base.py

```diff
@@ -25,7 +25,7 @@ def get_connection_params(self):
         )
         return conn_params
 
-    def create_cursor(self):
+    def create_cursor(self, name=None):
         # cursor_factory is a kwarg to connect() so restore create_cursor()'s
         # default behavior
-        return base.DatabaseWrapper.create_cursor(self)
+        return base.DatabaseWrapper.create_cursor(self, name=name)
```

Comment:
I guess something like this will do the trick:

    if django.VERSION >= (1, 11, 0):
        return base.DatabaseWrapper.create_cursor(self, name=name)
    else:
        return base.DatabaseWrapper.create_cursor(self)